.github/workflows/gradle.yml

```
@@ -0,0 +1,105 @@
+          for i in {1..30}; do
+            if docker exec postgres pg_isready -U walkit -d walkit -q; then
+              echo "PostgreSQL is up!"
+              break
+            fi
+            if [ "$i" -eq 30 ]; then
+              echo "PostgreSQL did not become ready in time" >&2
+              exit 1
+            fi
+            sleep 2
+          done
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)